csharp/src/Ice/OutputStream.cs

```diff
@@ -1577,7 +1577,7 @@ internal void WriteEndpoint(Endpoint endpoint)
             {
                 Position startPos = _tail;
                 int sizeLength = OldEncoding ? 4 : 2;
-                if (endpoint.Protocol == Protocol.Ice1 && endpoint is OpaqueEndpoint opaqueEndpoint)
+                if (endpoint is OpaqueEndpoint opaqueEndpoint)
```

Comment:
Why do we have this check before?

Reply:
I think it was a small mistake, since an opaque endpoint is ice1-only.